src/components/playbackSettings/playbackSettings.template.html

```diff
@@ -146,6 +146,14 @@ <h2 class="sectionTitle">
             </div>
         </div>
 
+        <div class="checkboxContainer checkboxContainer-withDescription">
```

Comment:
I would place it before `fldExternalPlayer`. When the latter is displayed, the order is odd: NextVideoOverlay, ExternalPlayer, DisplayRatings (_for next video overlay_).